doc/user/truffleruby-additions.md

```diff
@@ -52,6 +52,44 @@ TruffleRuby provides these non-standard methods and classes that provide additio
 
 * `AtomicReference` is marshalable.
 
+### Concurrent Maps
+
+`TruffleRuby::ConcurrentMap` is a key-value data structure, like a `Hash` and using `#hash` and `#eql?` to compare keys and identity to compare values. Unlike `Hash` it is unordered. All methods on `TruffleRuby::ConcurrentMap` are thread-safe but should have higher concurrency than a fully syncronized implementation. It is intended to be used by gems such as Concurrent Ruby - please use via this gem rather than using directly in most cases.
+
+* `map = TruffleRuby::ConcurrentMap.new([initial_capacity: 1024], [load_factor: 0.5])`
```

Comment:
Are these the default values, or some example?
I wouldn't show any specific value here, as people might otherwise copy-paste (and those defaults seem not good defaults).
`map = TruffleRuby::ConcurrentMap.new(options = nil)` would also be enough IMHO, we don't want people to call this manually anyway. Just specifying we support the options that concurrent-ruby supports seems enough.

Reply:
Done.

Reply:
Done.